recipe/meta.yaml

```diff
@@ -44,14 +45,14 @@ test:
     - zfit.minimizers
     - zfit.models
     - zfit.util
-    - zfit.ztf
+    - zfit.z
```

Comment:
```suggestion
     - zfit.ztf
```

Reply:
Yes, I meant to close it, I will apply the changes in the new version.